### PR TITLE
Feat/info bird

### PR DESCRIPTION
**Pull request by CBradsen**
## 🛠️ Description of this branch

* Sets up BirdInfo component to display more details about a single bird

- [X] New Feature
- [ ] Fix
- [ ] Documentation
- [ ] Tests

## 👀 Checks
- [ ] When the project is run locally, the terminal shows no errors or warnings
- [ ] Breaks nothing
- [ ] Breaks some things & Creates new issues
- [ ] Code in this branch has been tested  
- [X] Forgot to write new tests for changes in this code, but it is added to the project board as a ticket. 
- [ ] Code is DRY, reusable, follows SRP and trends toward function purity 
- [ ] I've updated the project board to reflect changes, add issues, or tickets

### CODE REVIEW

File: package.json
Comment (sdmisra) [inline package.json:1]: We can change line 22 in the future if we find that command awkward, otherwise, I am just glad we have the command in at all!

**Review by cadechaney — APPROVED** [review]
Good starting layout. Next we need to hook this component up to a router so it will render by itself.

**Review by sdmisra — APPROVED** [review]
We're real close to our MVP expectations, in rough functionality, with some time ahead of us to polish it up!

**Review by jmartin777 — APPROVED** [review]
Thanks for  putting this together asynced with the routing! Excited to connect to this and see the display

**Pull request merged**